### ISSUE TITLE
Architecture Diagram

### ISSUE THREAD

**Issue by danielballan** [bluesky/tiled#671]
For comment, to be added to the architecture page

![image](https://github.com/bluesky/tiled/assets/2279598/bb6df02a-c2a6-43da-b395-de6762362d72)

**Comment by padraic-shafer** [comment]
Maybe add web browser to the left-most orange box?  "e.g. web browser, curl, HTTPie"

Meaning the raw output, rather than the tiled web UI.  It can be a simple baseline debugger / get the metadata...but it can open up a can of worms with auth involved.  Thus I'm a bit conflicted about recommending it.

**Comment by danielballan** [comment]
> Do the ASGI dotted lines at the top make use of a httpx AsyncClient / starlette TestClient (or something similar)?

Yes, thought I might put it the other way around:  httpx AsyncClient / starlette TestClient make use of ASGI.

I have the same conflicting feeling about web browsers. It tells a nice story, but may get messy with auth.
